### ISSUE TITLE
Add a general xAPI validation option for the command `validate` 

### ISSUE THREAD

**Issue by wilbrdt** [openfun/ralph#388]
## Feature Request

**Is your feature request related to a problem or unsupported use case? Please describe.**
Right now, we are validating xAPI statements against our xAPI models implemented in Ralph.

**Describe the solution you'd like**
We could imagine that we would want to validate xAPI statements for profiles not defined in Ralph (we won't ever have all the profiles implemented in Ralph).
We could make a general xAPI validation, which checks that `actor`,  `id`, `object` and `verb` are present.
We could use the `LaxStatements` pydantic model.
The command could be: 
`bin/ralph validate -f xapi` to validate generic xAPI format, and `bin/ralph validate -f xapi --full` to validate against profiles? 
or
`bin/ralph validate -f xapi --generic` and `bin/ralph validate -f xapi --full`

**Discovery, Documentation, Adoption, Migration Strategy**
Of course, documentation would need an update.

**Comment by quitterie-lcs** [comment]
Why not using the `BaseXapiStatement` model rather than the `LaxStatements`

**Comment by wilbrdt** [comment]
You're right, it's preferable to use `BaseXapiStatement` instead of `LaxStatements` that was created for a performance purpose
